### DISCUSSION TITLE
Documentations and examples

### DISCUSSION THREAD

**Discussion by BaGRoS** [subzeroid/aiograpi-rest#56] [Q&A]
Hi

I will use [Lamadava SaaS](https://lamadava.com/), but I don't know how exactly.
Where I can download some examples? I asked support by the email address on the webpage but no answer yet.
I don't want, pay for something with zero support.

Many thanks,
BaGRoS

**Comment by adw0rd** [comment]
https://api.lamadava.com/docs

This is all the documentation, there is absolutely everything you need
